### ISSUE TITLE
piece hides below neighbour during hover

### ISSUE THREAD

**Issue by moonpatel** [moonpatel/ChessHub#7]
![image-hover-chess](https://github.com/moonpatel/ChessHub/assets/95487062/265e4d8b-4b93-49fe-ae08-31ebf8790bf8)
The selected piece hides behind the neighbor piece when it is hovered over other pieces coming after it on the board. There maybe some CSS issue in Piece.jsx or Cell.jsx.

**Comment by WIZN0DE** [comment]
Hi, @moonpatel can I work on this?

**Comment by moonpatel** [comment]
Yeah sure, you can work on this, @slayeh17

**Comment by WIZN0DE** [comment]
hey @moonpatel, I am using docker to run the application, and when I sign up or login I get a "Something went wrong" message.

this is what shows when I run ` sudo docker-compose up `

```` Starting chesshub_backend_1  ... done
Starting chesshub_frontend_1 ... done
Attaching to chesshub_frontend_1, chesshub_backend_1
backend_1   | 
backend_1   | > backend-api@1.0.0 start
backend_1   | > node app.js
backend_1   | 
frontend_1  | 
frontend_1  | > chess-@0.0.0 dev
frontend_1  | > vite --host
frontend_1  | 
backend_1   | Listening on server 8080
frontend_1  | 
frontend_1  |   VITE v4.4.7  ready in 264 ms
frontend_1  | 
backend_1   | Error in connecting to database
frontend_1  |   ➜  Local:   http://localhost:5173/
frontend_1  |   ➜  Network: http://172.18.0.2:5173/
backend_1   | node:internal/process/promises:289
backend_1   |             triggerUncaughtException(err, true /* fromPromise */);
backend_1   |             ^
backend_1   | 
backend_1   | Error: spawn engine/stockfish16.exe ENOENT
backend_1   |     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
backend_1   |     at onErrorNT (node:internal/child_process:484:16)
backend_1   |     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
backend_1   |   errno: -2,
backend_1   |   code: 'ENOENT',
backend_1   |   syscall: 'spawn engine/stockfish16.exe',
backend_1   |   path: 'engine/stockfish16.exe',
backend_1   |   spawnargs: []
backend_1   | }
backend_1   | 
backend_1   | Node.js v20.8.0
chesshub_backend_1 exited with code 1

**Comment by moonpatel** [comment]
> hey @moonpatel, I am using docker to run the application, and when I sign up or login I get a "Something went wrong" message.
> 
> this is what shows when I run `sudo docker-compose up`
> 
> ```
> Starting chesshub_frontend_1 ... done
> Attaching to chesshub_frontend_1, chesshub_backend_1
> backend_1   | 
> backend_1   | > backend-api@1.0.0 start
> backend_1   | > node app.js
> backend_1   | 
> frontend_1  | 
> frontend_1  | > chess-@0.0.0 dev
> frontend_1  | > vite --host
> frontend_1  | 
> backend_1   | Listening on server 8080
> frontend_1  | 
> frontend_1  |   VITE v4.4.7  ready in 264 ms
> frontend_1  | 
> backend_1   | Error in connecting to database
> frontend_1  |   ➜  Local:   http://localhost:5173/
> frontend_1  |   ➜  Network: http://172.18.0.2:5173/
> backend_1   | node:internal/process/promises:289
> backend_1   |             triggerUncaughtException(err, true /* fromPromise */);
> backend_1   |             ^
> backend_1   | 
> backend_1   | Error: spawn engine/stockfish16.exe ENOENT
> backend_1   |     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
> backend_1   |     at onErrorNT (node:internal/child_process:484:16)
> backend_1   |     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
> backend_1   |   errno: -2,
> backend_1   |   code: 'ENOENT',
> backend_1   |   syscall: 'spawn engine/stockfish16.exe',
> backend_1   |   path: 'engine/stockfish16.exe',
> backend_1   |   spawnargs: []
> backend_1   | }
> backend_1   | 
> backend_1   | Node.js v20.8.0
> chesshub_backend_1 exited with code 1
> ```

Make sure to set the .env file based on the .env.example templaye then build again.
Also I have made some changes in docker-compose.yml file so pull before building again.

**Comment by WIZN0DE** [comment]
Okay thanks 👍

**Comment by DeepRahangdale** [comment]
Hi, @moonpatel can I work on this as well?

**Comment by moonpatel** [comment]
Yeah, @DeepRahangdale you can work on this. @slayeh17 is already working on this issue so I will merge whoever first makes a pr

**Comment by WIZN0DE** [comment]
Hey, @moonpatel I am unassigning myself, can't work on this unfortunately because I don't have much experience with React yet.

**Comment by moonpatel** [comment]
> Hey, @moonpatel I am unassigning myself, can't work on this unfortunately because I don't have much experience with React yet.

Sure, @slayeh17 you can try looking for some other good first issues or try out the hosted application to find some bugs

**Comment by moonpatel** [comment]
> Hi, @moonpatel can I work on this as well?

Hey @DeepRahangdale are you still working on this?
Would you like me to assign this to you?

**Comment by DeepRahangdale** [comment]
@moonpatel haven't started yet... will work on homepage instead of this!!!!

**Comment by moonpatel** [comment]
Ok tell me afterwards if you want to work on this

> @moonpatel haven't started yet... will work on homepage instead of this!!!!
